modules/wallet/persist.go

```diff
@@ -103,6 +110,29 @@ func (w *Wallet) initSettings() error {
 	return w.loadSettings()
 }
 
+// openDB loads the set database and populates it with the necessary buckets.
+func (w *Wallet) openDB(filename string) (err error) {
+	w.db, err = persist.OpenDatabase(dbMetadata, filename)
+	if err != nil {
+		return err
+	}
+	// initialize the database
+	err = w.db.Update(func(tx *bolt.Tx) error {
+		buckets := [][]byte{
+			bucketHistoricOutputs,
+			bucketHistoricClaimStarts,
+		}
+		for _, b := range buckets {
+			_, err := tx.CreateBucketIfNotExists(b)
+			if err != nil {
+				return err
+			}
+		}
+		return nil
+	})
+	return err
```

Comment:
This blob of code here is likely to grow. It's fine for now but should keep in mind what the structure will be turning into. Ideally, any boltdb code is written such that it's easy to reason about the performance impact.